kurobako/solver/optuna.py

```diff
@@ -12,9 +12,12 @@
 
 from kurobako import problem
 from kurobako import solver
+import logging
 
 _optuna_logger = optuna.logging.get_logger(__name__)
-
+_optuna_logger.setLevel(optuna.logging.INFO)
+console = logging.StreamHandler()
+_optuna_logger.addHandler(console)
```

Comment:
Could you explain the necessity of this change?

Reply:
In my understanding, handler and the log level are usually set in the user code, not library code.
I don't think it is a good practice to show log messages.